ee/tabby-webserver/src/service/cron/db.rs

```diff
@@ -0,0 +1,58 @@
+use anyhow::Result;
+use tokio_cron_scheduler::Job;
+use tracing::error;
+
+use crate::{schema::JobRunResult, service::db::DbConn};
+
+pub async fn refresh_token_job(db_conn: DbConn) -> Result<Job> {
+    // job is run every 2 hours
+    let job = Job::new_async("0 0 1/2 * * * *", move |_, _| {
+        let db_conn = db_conn.clone();
+        Box::pin(async move {
+            let res = db_conn.delete_expired_token().await;
+            if let Err(e) = res {
+                error!("failed to delete expired token: {}", e);
+            }
+        })
+    })?;
+
+    Ok(job)
+}
+
+pub async fn repository_job(db_conn: DbConn, job_name: String, schedule: &str) -> Result<Job> {
```

Comment:
`db.rs` here means it's about jobs doing db maintenance.

move to individual file, or just `mod.rs`.